GAE/src/com/gallatinsystems/common/util/MD5Util.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (C) 2010-2012 Stichting Akvo (Akvo Foundation)
+ *  Copyright (C) 2010-2018 Stichting Akvo (Akvo Foundation)
```

Comment:
Small note: following the strict convention this would be `2010-2012,2028`

Reply:
are we following the _strict convention_?, if so, we need to fix *lots* of copyright headers

Reply:
e.g. according to `git log` this file was modified in 2011,2012,2014,2018

     git log --date=short GAE/src/com/gallatinsystems/common/util/MD5Util.java | awk '/^Date:/ {print $2}'

I guess that something similar applies to several files

Reply:
This of course applies to all files but following the boy scout rule, this is the file you were working on at the moment and can actually do something about without disrupting too much else.